IllegalType/all-examples-in-one/config.xml

```diff
@@ -53,15 +53,13 @@
         </module>
 
         <module name="IllegalType">
-            <property name="id" value="example7"/>
-            <property name="id" value="example7"/>
+            <property name="id" value="IllegalTypeOptionalAsField"/>
```

Comment:
@romani i have added the new code, can you help me how to resolve this?

Reply:
I can use a hack but as mentioned you don't want it. unfortunately, this time it takes the moduleName as value for new files.